zerver/webhooks/beeminder/tests.py

```diff
@@ -3,7 +3,7 @@
 from typing import Text
 from zerver.lib.test_classes import WebhookTestCase
 current_time = float(t.time())
-time = (float(1515759100) - current_time)/float(3600)
+time = (float(1517759100) - current_time)/float(3600)
```

Comment:
Again what's the point behind this? :)

Reply:
@Rishabh570: Again, one thing I don't understand is why do you need that hardcoded value `1517759100`?

Reply:
@eeshangarg I tried making a function and passing a payload as the argument...

Reply:
@Rishabh570: I see what you mean now! This is not a big deal so this is fine for testing! :)